src/libs/ReportUtils.js

```diff
@@ -1970,6 +1970,28 @@ function getChatByParticipants(newParticipantList) {
     });
 }
 
+/**
+ * Attempts to find a report in onyx with the provided email list of participants. Does not include threads
+ * This is temporary function while Migrating from PersonalDetails to PersonalDetailsList
+ *
+ * @deprecated - use getChatByParticipants()
+ *
+ * @param {Array} participantsloginList
+ * @returns {Array|undefined}
+ */
+function getChatByParticipantsByLoginList(participantsloginList) {
```

Comment:
```suggestion
function getChatByParticipantsLoginList(participantsloginList) {
```

Reply:
I think it is fine. As we're passing loginList.
The proposed name can be confusing with `getChatByParticipants()`